### PR TITLE
voting: fix some spelling issues

### LINKED ISSUE #4584: Punctuation (Wording) issues (English) for translation with Weblate - related to [6487]
**URL** 
**user:** 
**expected behaviour:** When translating from English into German with Webtale, according to Webtale the punctuation (including spaces) of English and German should match.
**behaviour:** Some English sentences in the code have one too many spaces or one too few full stops. With Weblate, for example, one too many spaces is automatically included in the translation. 
**important screensize:**
**device & browser:** 
**Comment/Question:** 

- [ ]  **String:** meinberlin/apps/votes/templates/meinberlin_votes/voting_dashboard.html:7 
-> One Space too many after the sentence _"For voting in the third phase, all participants need their own voting code. Here you can generate up to %(tokens_per_module)s codes. Depending on the number this may take a few minutes."_

- [ ] **String:** meinberlin/apps/votes/templates/meinberlin_votes/voting_code_dashboard.html:7 
-> Missing point at the end of the sentence _"For voting in the third phase, all participants need their own voting code. Here you can download the codes in packages of %(export_size)s per excel file. To generate the codes, please contact %(contact_email)s"_

### PR DESCRIPTION
**Pull request by goapunk**
fixes #4584

### CODE REVIEW

File: locale/de_DE/LC_MESSAGES/djangojs.po [hunk 1]
```diff
@@ -82,64 +82,44 @@ msgstr ""
 "er nicht unseren Diskussionsregeln (Netiquette) entspricht."
 
 #: adhocracy4/comments/static/comments/Comment.jsx:96
-#: adhocracy4/comments_async/static/comments_async/comment.jsx:296
 #: adhocracy4/comments/static/comments/Comment.jsx:96
-#: adhocracy4/comments_async/static/comments_async/comment.jsx:296
```
Comment (phillimorland) [inline locale/de_DE/LC_MESSAGES/djangojs.po:87]: @goapunk there are a lot of deleted translation ref files?!

Comment (goapunk) [reply locale/de_DE/LC_MESSAGES/djangojs.po:87]: all those changes are coming from `make po`, or is there anything else which needs to be done?

Comment (phillimorland) [reply locale/de_DE/LC_MESSAGES/djangojs.po:87]: no but we have had issues in the past where js file references are deleted, we do still need the references though, I will try running make po on your PR and check

Comment (phillimorland) [reply locale/de_DE/LC_MESSAGES/djangojs.po]: @goapunk I pushed mine which brings it all back, shall I just merge the commits?

Comment (goapunk) [reply locale/de_DE/LC_MESSAGES/djangojs.po]: yeah, thanks.

**Pull request merged**